locales/fr.yml

```diff
@@ -0,0 +1,42 @@
+---
+fr:
+  site_name: "Sécurité sans Frontières"
+  follow_us: "Suivez-nous sur Twitter : %{twitter}"
+  request_assistance: "Demander de l'aide"
+  intro_paragraphs: "Depuis l'avènement de l'ère digitale, ceux qui dénoncent
+    la corruption, luttent contre le racisme et défendent la justice et la
+    démocratie sur Internet sont plus que jamais menacés.<br /><b>Sécurité sans
```

Comment:
Not sure either, but it's consistent with the original text. If it's changed here, it should be changed everywhere.